velox/functions/prestosql/tests/CastBaseTest.h

```diff
@@ -127,6 +126,51 @@ class CastBaseTest : public FunctionBaseTest {
     assertEqualVectors(wrapInDictionary(indices, expected), result);
   }
 
+  /**
+   * @tparam From Source type for cast.
+   * @tparam To Destination type for cast.
+   * @param typeString Cast type in string.
+   * @param input Input vector of type From.
+   * @param expectedResult Expected output vector of type To.
+   */
+  template <typename TFrom, typename TTo>
+  void testCast(
+      const std::string& typeString,
+      std::vector<std::optional<TFrom>> input,
+      std::vector<std::optional<TTo>> expectedResult,
+      bool tryCast = false,
```

Comment:
This is needed when test case wants to verify if error cases lead to null outputs.

Reply:
Can we remove tryCast parameter here and add another method: testTryCast. This will make tests a bit easier to read.